components/list/list.css

```diff
@@ -0,0 +1,58 @@
+.list-box {
+    width: 20rem;
+    max-width: 92%;
+    border-radius: 5px;
+    box-shadow: 0px 0px 1px 1px var(--bg-secondary-color);
```

Comment:
What I did is declare a variable for shadow color and use that. Example below:
 --shadow-color-light: rgb(23, 23, 23, 0.5);
--shadow-color-dark: rgb(23, 23, 23, 3.5);

Declare with other variables in root. Please ask me if something is not clear.